zigpy/zcl/clusters/__init__.py

```diff
@@ -15,8 +16,8 @@
 )
 from .. import Cluster
 
-CLUSTERS_BY_ID: dict[int, Cluster] = {}
-CLUSTERS_BY_NAME: dict[str, Cluster] = {}
+CLUSTERS_BY_ID: typing.Dict[int, Cluster] = {}
```

Comment:
Add `from __future__ import annotations` at the very top of the file, it will allow built-in types like `dict` to be used in annotations on older versions of Python.

Reply:
Good, I was wondering what the trick was to get 'dict' working.